src/helpers.php

```diff
@@ -29,24 +29,38 @@ function tenant($key = null)
     }
 }
 
-if (! \function_exists('tenant_asset')) {
+if (! function_exists('tenant_asset')) {
     /** @return string */
     function tenant_asset($asset)
     {
         return route('stancl.tenancy.asset', ['path' => $asset]);
     }
 }
 
-if (! \function_exists('global_asset')) {
+if (! function_exists('global_asset')) {
     function global_asset($asset)
     {
         return app('globalUrl')->asset($asset);
     }
 }
 
-if (! \function_exists('global_cache')) {
+if (! function_exists('global_cache')) {
     function global_cache()
     {
         return app('globalCache');
     }
 }
+
+if (! function_exists('tenant_route')) {
+    function tenant_route(string $route, array $parameters = [], string $domain = null): string
```

Comment:
Strict mode improvement: 

Please specify that `$domain` can be nullable. 

```php
..., ?string $domain = null)
```

Reply:
This is not needed. Setting the default value to null automatically makes the type nullable.